content/docs/command-reference/exp/pull.md

```diff
@@ -11,7 +11,7 @@ usage: dvc exp pull [-h] [-q | -v] [-f] [--no-cache]
                     git_remote experiment
 
 positional arguments:
-  git_remote          Git remote name or repo URL
+  git_remote          Git remote name or valid Git URL
```

Comment:
```suggestion
  git_remote    Optional Git remote name or Git URL (SSH auth only)
```

Reply:
The Git remote is not optional for `dvc exp pull`.